src/Frontend/Page/Overview.hs

```diff
@@ -175,6 +175,7 @@ instance ToHtml Tabs where
 -- * idea lists
 
 data ListItemIdeaContext  -- TODO: rename to 'ListLocation'?  'Context' is used for too many things.
+                          -- 'Position'?  i think my favorite so far would be `WhatList`.  ~~mf
```

Comment:
For me `WhatListPage` is a question. :) This is the first time in my life, I see a question as a type name. Which is kind of make sense, if we see this as an information about the context, which is represented as an extra paramter, which is isomorph to a pair. Also we push this info from a call, like the inversion of control from the client code to the function call, where inversion could mean the dual of the function call, which basically a dual of a function which is a pair. Does it hurt? :D

Summary: I kind of like it.